compose/reference/push.md

```diff
@@ -31,5 +31,5 @@ services:
 
   service2:
     build: .
-    image: youruser/yourimage  # goes to youruser DockerHub registry
+    image: youruser/yourimage  # goes to your user DockerHub registry
```

Comment:
Wondering if this was a correct change (but I can see the confusion) I think this was meant to spell `youruser` as that's the example username on Docker Hub.

Perhaps we should change to `your-user` or `your-username` or `hub-username`

We should likely change `DockerHub` to `Docker Hub` though 😅

Reply:
Agree.  `your-username` might be better to avoid confusion. I'll fix this now.